gradle.properties

```diff
@@ -1 +1,2 @@
 org.gradle.parallel=true
+includeProviders=azure,cassandra,pipelines,gcs,oracle,redis,s3,swift
```

Comment:
Could you also add some text in the README with instructions on how to only build with specific providers?

Reply:
Done!